wrapper/src/main/java/software/amazon/jdbc/DataSourceConnectionProvider.java

```diff
@@ -114,6 +116,14 @@ public Connection connect(
       final @NonNull HostSpec hostSpec,
       final @NonNull Properties props)
       throws SQLException {
+    final String strategy = ReadWriteSplittingPlugin.READER_HOST_SELECTOR_STRATEGY.getString(props);
+    if (strategy != null && strategy.equals("roundRobin")) {
```

Comment:
Nit: when using `.equals()` for strings, its safe practice to have the string literal on the left side incase whatever string variable that is being compared, is a null. 
ex) `"a".equals(b)` is safer than `b.equals("a")